support/config/deployment_test.go

```diff
@@ -392,3 +394,102 @@ func TestResourceRequestOverrides(t *testing.T) {
 		})
 	}
 }
+
+func TestSetSafeToEvictLocalVolumesAnnotation(t *testing.T) {
```

Comment:
This should be call TestApplyTo which the function you are testing, unless you had extracted the particular annotation logic into its own function SetSafeToEvictLocalVolumesAnnotation which is not the case